src/components/Player/TableFilterForm/index.jsx

```diff
@@ -204,6 +204,15 @@ class TableFilterForm extends React.Component {
               strict
               limit={1}
             />
+            <FormField
+              name="party_size"
+              label={strings.filter_party_size}
+              dataSource={data.partySize}
+              formSelectionState={formSelectionState}
+              history={history}
+              strict
+              limit={5}
```

Comment:
I think this should be 1

Reply:
But what if someone wants all games with party size 2 and party size 1 for example?

Reply:
Nevermind, that doesn't work probably.